### PR TITLE
qbittorrent: vuetorrent initial commit

### PR DESCRIPTION
**Pull request by peanutbother**
This adds vuetorrent mod to official supported qbittorrent mods.

### CODE REVIEW

**Comment by peanutbother** [comment]
This should be complete, but I'll let you look over it, if I missed something

File: README.md [hunk 1]
```diff
@@ -1,17 +1,8 @@
-# Rsync - Docker mod for openssh-server
+# VueTorrent WebUi - Docker mod for qBittorrent
 
-This mod adds rsync to openssh-server, to be installed/updated during container start.
+This mod adds [VueTorrent](https://github.com/WDaan/VueTorrent), to be installed to [qBittorrent](https://github.com/linuxserver/docker-qbittorrent/) during container start.
 
-In openssh-server docker arguments, set an environment variable `DOCKER_MODS=linuxserver/mods:openssh-server-rsync`
+In qBittorrent docker arguments, set an environment variable `DOCKER_MODS=linuxserver/vuetorrent` to enable.
```
Comment (drizuid) [inline README.md:5]: `DOCKER_MODS=linuxserver/vuetorrent` should be `DOCKER_MODS=linuxserver/mods:qbittorrent-vuetorrent` as shown in the example.

File: README.md [hunk 2]
```diff
@@ -1,17 +1,8 @@
...
-
-* Fork the repo, create a new branch based on the branch `template`.
-* Edit the `Dockerfile` for the mod. `Dockerfile.complex` is only an example and included for reference; it should be deleted when done.
-* Inspect the `root` folder contents. Edit, add and remove as necessary.
-* Edit this readme with pertinent info, delete these instructions.
-* Finally edit the `.github/workflows/BuildImage.yml`. Customize the build branch, and the vars for `BASEIMAGE` and `MODNAME`.
-* Ask the team to create a new branch named `<baseimagename>-<modname>`. Baseimage should be the name of the image the mod will be applied to. The new branch will be based on the `template` branch.
-* Submit PR against the branch created by the team.
+If adding multiple mods, enter them in an array separated by `|`, such as 
+> `DOCKER_MODS=linuxserver/vuetorrent|linuxserver/mods:other-mod`
```
Comment (drizuid) [inline README.md:8]: `DOCKER_MODS=linuxserver/vuetorrent|linuxserver/mods:other-mod` should be `DOCKER_MODS=linuxserver/mods:qbittorrent-vuetorrent|linuxserver/mods:other-mod` as shown in the example.

**Review by drizuid — CHANGES_REQUESTED** [review]
the same issues pointed out by aptalca 18 days ago still remain.  The readme also says this only installs, it won't update on a new release? Have you tested this on x86_64 and any arm variants?

**Comment by peanutbother** [comment]
forgot to push my commit 😅
I tested with x64 and arm64.

The script downloads latest release of `vuetorrent`, saves release data and compares it at container start with current release, so if there's a new version `vuetorrent` gets updated.

File: README.md [hunk 3]
```diff
@@ -1,8 +1,10 @@
 # VueTorrent WebUi - Docker mod for qBittorrent
 
-This mod adds [VueTorrent](https://github.com/WDaan/VueTorrent), to be installed to [qBittorrent](https://github.com/linuxserver/docker-qbittorrent/) during container start.
+This mod adds [VueTorrent](https://github.com/WDaan/VueTorrent), to be installed to [qBittorrent](https://github.com/linuxserver/docker-qbittorrent/) during container start.
```
Comment (drizuid) [inline README.md:3]: please change installed to installed/updated as you confirmed it also updates

Comment (peanutbother) [reply README.md:3]: If changing to installed/updated; should I remove the second sentence or change it in any way?

Comment (drizuid) [reply README.md:3]: I would so it follows the standard the rest use.

**Review by drizuid — APPROVED** [review]

File: root/etc/cont-init.d/98-vuetorrent [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+NEEDS_UPDATE=false
+CURRENT_RELEASE=$(curl -s https://api.github.com/repos/WDaan/VueTorrent/releases/latest | jq -r ".assets[]")
+
+# ===============================================
+install_deps() {
+    if ! [[ -x "$(command -v jq)" ]]; then
+        echo '-------------------------'
+        echo '| Installing jq package |'
+        echo '-------------------------'
+        apt-get update &&
```
Comment (aptalca) [inline root/etc/cont-init.d/98-vuetorrent:13]: Please move this out to a separate init script called `95-apt-get` like here: https://github.com/linuxserver/docker-mods/blob/code-server-awscli/root/etc/cont-init.d/95-apt-get

File: root/etc/cont-init.d/97-apt-get [hunk 5]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/with-contenv bash
+
+if ! [[ -x "$(command -v jq)" ]]; then
+    echo '-------------------------'
+    echo '| Installing jq package |'
+    echo '-------------------------'
+    apt-get update &&
+    apt-get install -y jq
+fi
```
Comment (aptalca) [inline root/etc/cont-init.d/97-apt-get:9]: this file should be named `95-apt-get` and it should only have `apt-get update` like this one here: https://github.com/linuxserver/docker-mods/blob/code-server-golang/root/etc/cont-init.d/95-apt-get

**Comment by aptalca** [comment]
closing due to inactivity. If you want to reopen a PR, ask us to create the branch first

**Pull request closed without merging**